internal/app/recordtester/recordtester_app.go

```diff
@@ -390,18 +456,12 @@ func (rt *recordTester) getIngestInfo() (*livepeer.Ingest, error) {
 	return &ingests[0], nil
 }
 
-func (rt *recordTester) doOneHTTPStream(fileName, streamName, broadcasterURL string, testDuration time.Duration, stream *livepeer.CreateStreamResp) error {
-	var session *livepeer.CreateStreamResp
+func (rt *recordTester) doOneHTTPStream(fileName, streamName, broadcasterURL string, testDuration time.Duration, stream *livepeerAPI.CreateStreamResp) error {
+	var session *livepeerAPI.CreateStreamResp
 	var err error
 	apiTry := 0
 	for {
-		session, err = rt.lapi.CreateStream(livepeer.CreateStreamReq{
-			Name:                streamName,
-			ParentID:            stream.ID,
-			Profiles:            standardProfiles,
-			Record:              true,
-			RecordObjectStoreId: rt.recordObjectStoreId,
-		})
```

Comment:
My bad on creating this simpler `CreateStream` function only on `stream-tester` and not replicating the change on `go-api-client`. I believe it provides a better and simpler interface (instead of multiple functions with cryptic names like "ex" "ex2", etc).

So I'd say that `go-api-client` should be the one migrating to provide this single `CreateStream` function instead, not the other way around. WDYT?